src/pages/Detail/ProductInfo/Precautions/Precautions.js

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import Styled from 'styled-components';
+
+const Precautions = () => {
+  return (
+    <>
+      <PrecautionsModal>
+        <i class="fas fa-exclamation-circle"></i>
+        <div className="GuideText">
+          <h3>판매 거래 주의사항</h3>
+          <p>반드시 보유한 상품만 판매하세요.</p>
+        </div>
+      </PrecautionsModal>
+      <PrecautionsAccordion>
+        <h3>구매 전 꼭 확인해주세요!</h3>
+        <div className="btns">
+          <div className="btnName">배송 기간 안내</div>
+          <div className="periodGuideParagraph">
+            <strong>
+              KREAM은 최대한 빠르게 모든 상품을 배송하기 위해 노력하고 있습니다.
+              배송 시간은 판매자가 검수를 위하여 상품을 검수센터로 보내는 속도에
+              따라 차이가 있습니다.
+            </strong>
+            <ul>
+              <li>
+                <p>
+                  - 거래가 체결된 시점부터 48시간(일요일•공휴일 제외) 내에
+                  판매자가 상품을 발송해야 하며, 통상적으로 발송 후 1-2일 내에
+                  KREAM 검수센터에 도착합니다.
+                </p>
+              </li>
```

Comment:
```suggestion
              <li>
                  - 거래가 체결된 시점부터 48시간(일요일•공휴일 제외) 내에
                  판매자가 상품을 발송해야 하며, 통상적으로 발송 후 1-2일 내에
                  KREAM 검수센터에 도착합니다.
              </li>
```
해당 부분들 p태그로 감싸지 않고 바로 사용하셔도 됩니다!